src/engraving/libmscore/rest.cpp

```diff
@@ -188,7 +188,7 @@ bool Rest::acceptDrop(EditData& data) const
         ) {
         return true;
     }
-    return type != ElementType::SLUR && e->isSpanner();
+    return type != ElementType::SLUR && type != ElementType::TIE && e->isSpanner();
```

Comment:
This is the same as my #8818

Reply:
Except for a comment:
`// prevent 'hanging' slurs, avoid crash on tie`

Reply:
> It contains multiple commits that don't belong here. Also fails the vtests?

Something's really odd with one of my local repos, that happens quite a lot.
The vtests is something random, and there's no obvious way to re-run (other than pushing more commits)

Reply:
@Jojo-Schmitz

Your PR has been merged

Reply:
But this diff doesn't show it...

Reply:
> Something's really odd with one of my local repos, that happens quite a lot.
May need to clean your repo, `git gc` or some such?

> The vtests is something random, and there's no obvious way to re-run (other than pushing more commits)
`git commit --amend; git push -f` is what I use in sich cases.